### PR TITLE
Refactor#2

### PR DESCRIPTION
**Pull request by harshavardhan-sureify**

### CODE REVIEW

**Review by rameshguptha-sureify — APPROVED** [review]

**Review by maniknr-sureify — APPROVED** [review]

**Review by rameshguptha-sureify — APPROVED** [review]

File: src/components/admin/AllUsers.jsx [hunk 1]
```diff
@@ -295,7 +309,7 @@ const AllUsers = () => {
                     </Box>
                     <Box sx={{ typography: "subtitle2" }}>
                         <Typography variant="subtitle2">
-                            Are you sure want to delete the user {" "}
+                            Are you sure want to delete the user{" "}
```
Comment (abhishekdundigalla-sureify) [inline src/components/admin/AllUsers.jsx:312]: remove {" "}

**Review by abhishekdundigalla-sureify — CHANGES_REQUESTED** [review]
Please check comments

File: src/components/features/ActiveTab.reducer.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { createSlice } from "@reduxjs/toolkit";
 
 const initialState = {
-    activeTab: "Active",
+    activeTab: localStorage.getItem("activeTab") ?? "Active",
```
[the remark this answers is not in the document]
Comment (harshavardhan-sureify) [reply src/components/features/ActiveTab.reducer.js:4]: that is Nullish coalescing operator (??)
?? return the right side value it left is null

File: src/components/admin/AllUsers.jsx (hunk 1 above)
Comment (harshavardhan-sureify) [reply src/components/admin/AllUsers.jsx:312]: done

File: src/components/features/ActiveTab.reducer.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (abhishekdundigalla-sureify) [reply src/components/features/ActiveTab.reducer.js:4]: that is not a proper syntax to use in react, change it to ternary or logical and operator

**Review by abhishekdundigalla-sureify — APPROVED** [review]

**Pull request merged**